spectree/utils.py

```diff
@@ -91,6 +91,11 @@ def parse_request(func: Any) -> Dict[str, Any]:
             "schema": {"$ref": f"#/components/schemas/{func.form}"}
         }
 
+    if hasattr(func, "body"):
+        content_items["text/plain"] = {
+            "schema": {"$ref": f"#/components/schemas/{func.body}"}
+        }
```

Comment:
What will the BaseModel look like? Can you provide an example? If possible, can you add some tests?

Reply:
@kemingy It is intended for text inputs like CSV etc. BaseModel is used to describe data in this model and it can be like this:

```
class FooBarPostBodyData(BaseModel):
    """
    CSV body for example for some import
    """
    __root__: str = Field(description="Data in CSV format - it consists of columns name, surname, ..., ..., ... .")
```

Reply:
@kemingy : ?

Reply:
Sorry for the late response.

I feel that if we need to create a special argument for each [MIME type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types), there will be endless arguments. By the way, they all belong to the request body. I wander if we can add an attribute to the request model, for example, `spec_mime_type` or something else. This should be able to handle this kind of feature. What do you think?

Reply:
Some old discussions here https://github.com/0b01001001/spectree/discussions/176

Reply:
Yes, it could be useful to process on one place for multiple mime-types.